lib/graphql/queries.js

```diff
@@ -900,6 +902,274 @@ export const getHostPendingApplicationsQuery = gql`
+          }
+        }
+        contributors(limit: $nbContributorsPerContributeCard) {
+          id
+          image
+          collectiveSlug
+          name
+          type
+        }
+      }
+      events(includePastEvents: true) {
+        id
+        slug
+        name
+        description
+        image
+        startsAt
+        endsAt
+        backgroundImageUrl(height: 208)
+        contributors(limit: $nbContributorsPerContributeCard, roles: [BACKER, ATTENDEE]) {
+          id
+          image
+          collectiveSlug
+          name
+          type
+        }
+        stats {
+          id
+          backers {
+            id
+            all
+            users
+            organizations
+          }
+        }
+      }
+      childCollectives {
+        id
+        slug
+        name
+        type
+        description
+        backgroundImageUrl(height: 208)
+        stats {
+          id
+          backers {
+            id
+            all
+            users
+            organizations
+          }
+        }
+        contributors(limit: $nbContributorsPerContributeCard) {
+          id
+          image
+          collectiveSlug
+          name
+          type
+        }
+      }
+      ...TransactionsAndExpensesFragment
+      updates(limit: 3, onlyPublishedUpdates: true) {
+        ...UpdatesFieldsFragment
+      }
+    }
+  }
+
+  ${fragments.TransactionsAndExpensesFragment}
+  ${fragments.UpdatesFieldsFragment}
+  ${fragments.ContributorsFieldsFragment}
+`;
+
+export const getTierPageQuery = gql`
+  query TierPage($tierId: Int!) {
+    Tier(id: $tierId) {
+      id
+      name
+      slug
+      description
+      longDescription
+      videoUrl
+      goal
+      currency
+      interval
+      endsAt
+
+      stats {
+        id
+        totalDonated
+        totalRecurringDonations
+        contributors {
+          id
+          all
+          collectives
+          organizations
+          users
+        }
+      }
+
+      collective {
+        id
+        slug
+        type
+        name
+        backgroundImage
+        settings
+        currency
+        isArchived
+        path
+        host {
+          id
+        }
+        stats {
+          id
+          updates
+          balance
+          transactions {
+            id
+            all
+          }
+        }
+        admins: members(role: "ADMIN") {
+          id
+          role
+          collective: member {
+            id
+            type
+            slug
+            name
+            image
+          }
+        }
+        parentCollective {
+          id
+          twitterHandle
+          image
+        }
+      }
+
+      contributors {
+        id
+        name
+        roles
+        isAdmin
+        isCore
+        isBacker
+        isFundraiser
+        since
+        description
+        publicMessage
+        collectiveSlug
+        totalAmountDonated
+        type
+        isIncognito
+        collectiveId
+      }
+    }
+  }
+`;
+
```

Comment:
I don't like this idea a lot. The reason is that it creates a weird relationship in my opinion. We'll have this little file making reference to two big files just to access the queries they use.

Also this will be a circular dependency. I don't know the ability of Next.js to handle it but I just like to play in the safe and not having them at all the is the safest option.

Reply:
Then feel free to create a new file in `components/collective-page/graphql/queries.js` (and the equivalent for tier page)

Reply:
I created:
`components/collective-page/graphql/queries.js` and
`components/tier-page/graphql/queries.js`